README.md

```diff
@@ -28,12 +28,12 @@ INSTALL
 Copy the statsd library from the .m2 folder to cassandra/lib.
 Add the following to your cassandra startup script:
 
-Copy the agent-1.1.jar to a new directory cassandra/plugins
+Copy the agent-1.2.jar to a new directory cassandra/plugins
 
 Change cassandra startup to add this agent. This can be done in
 a stock install by adding the following to /etc/default/cassandra:
```

Comment:
You are THE expert here. 

But, when I ran the tests yesterday, I added the following line conf/cassandra-env.sh. There is a lot of option tweaking there - including JMX settings.